### ISSUE TITLE
default Spectral coloramp in smudge plots not Colour Blind Friendly

### ISSUE THREAD

**Issue by ggrimes** [tbenavi1/smudgeplot#15]
The default brewer coloramp (get_default_col_ramp.R) in the smudge plots is very problematic for color blind individuals can you please change it to a perceptually uniform, qualitative, colour blind friendly palette like viridis (https://cran.r-project.org/web/packages/viridis/vignettes/intro-to-viridis.html).

```
> brewer.pal.info["Spectral",]
         maxcolors category colorblind
Spectral        11      div      FALSE
```

Thanks

**Comment by KamilSJaron** [comment]
Uf, I thought that Spectral is colorblind friendly. Sorry, my bad. I will update it.

**Comment by ggrimes** [comment]
Thanks for that.

Best wishes,

Graeme

**Comment by KamilSJaron** [comment]
Alright, we made two changes. 

- First we changed the transformation we use for the kmer frequencies from quadratic to log scale. To make sure that people get a good idea about the genome we now report also non-transformed smudgeplot.
- we use viridis palette

![mflo2_smudgeplot__smudgeplot_log10](https://user-images.githubusercontent.com/8181573/45959760-f1032d00-c01a-11e8-8576-ff0512c33da9.png)
![mflo2_smudgeplot__smudgeplot](https://user-images.githubusercontent.com/8181573/45959761-f1032d00-c01a-11e8-92a6-3eb513c86228.png)
